src/AbstractAlgebra.jl

```diff
@@ -1155,27 +1156,11 @@ function YoungTableau(part::Generic.Partition, fill::Vector{Int}=collect(1:part.
    Generic.YoungTableau(part, fill)
 end
 
-function number_field(a::Generic.Poly{Rational{BigInt}}, s::AbstractString, t = "\$"; cached = true)
+function number_field(a::Generic.Poly{Rational{BigInt}}, s::VarName, t = "\$"; cached = true)
```

Comment:
Stupid question, but: what is `t` here? Is this another "symbol"?

The documentation says:
> The string fields are currently ignored, but are reserved for future use.

But what's the point of that?! The arguments are completely unspecified, after all. Maybe they are matching something in Nemo or Hecke? But I don't see an argument `t` in the OSCAR docstrings for these functions either?

(This is of course not a blocker for this PR, I just noticed it while reviewing the PR)

Reply:
This function will never be called in OSCAR/Nemo/Hecke :)